lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -253,8 +255,20 @@ namespace :foreman_tasks do
         total = tasks.count
 
         tasks.each_with_index do |task, count|
-          File.open(File.join(tmp_dir, "#{task.id}.html"), 'w') { |file| file.write(PageHelper.pagify(renderer.render_task(task))) }
-          puts "#{count + 1}/#{total}"
+          begin
```

Comment:
You can drop this `begin` and matching `end` and indent the block body one level less